charts/growi/templates/deployment.yaml

```diff
@@ -3,50 +3,87 @@ kind: Deployment
 metadata:
   name: {{ include "growi.fullname" . }}
   labels:
-{{ include "growi.labels" . | indent 4 }}
+    {{- include "growi.labels" . | nindent 4 }}
 spec:
   replicas: {{ .Values.replicaCount }}
   selector:
     matchLabels:
-      app.kubernetes.io/name: {{ include "growi.name" . }}
-      app.kubernetes.io/instance: {{ .Release.Name }}
+      {{- include "growi.labels" . | nindent 6 }}
+  {{- with .Values.strategy }}
+  strategy:
+    {{- . | toYaml | nindent 4 }}
+  {{- end }}
   template:
     metadata:
       labels:
-        app.kubernetes.io/name: {{ include "growi.name" . }}
-        app.kubernetes.io/instance: {{ .Release.Name }}
-    spec:
-    {{- with .Values.imagePullSecrets }}
-      imagePullSecrets:
+        {{- include "growi.labels" . | nindent 8 }}
+        {{- with .Values.podLabels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+      annotations:
+        {{- with .Values.podAnnotations }}
         {{- toYaml . | nindent 8 }}
-    {{- end }}
+        {{- end }}
+    spec:
       containers:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           ports:
-            - name: http
-              containerPort: 80
-              protocol: TCP
+          - name: http
+            containerPort: 3000
+            protocol: TCP
           livenessProbe:
             httpGet:
-              path: /
+              path: /_api/v3/healthcheck
               port: http
+            initialDelaySeconds: 10
           readinessProbe:
             httpGet:
-              path: /
+              path: /_api/v3/healthcheck?connectToMiddlewares
               port: http
+            initialDelaySeconds: 10
+          env:
+          {{- with .Values.extraEnv }}
```

Comment:
The environment variables used in Deployment are described in ConfigMap, Secret, and please use `envFrom.configMapRef` and `secretRef` in here.